gulpfile.js

```diff
@@ -47,7 +47,7 @@ var filePath = {
     },
     styles: {
         src: './app/app.less',
-        watch: ['./app/app.less', './app/**/*.less']
+        watch: ['./app/app.less', './app/**/*.less', './app/components/**/*.less']
```

Comment:
I wouldn't think this is necessary, the ** should cover directories of directories and would include components.